<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddLocalDialog</class>
 <widget class="QDialog" name="AddLocalDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QLabel" name="label_Entity">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Notifications</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Reports are triggered by</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="combo_LocalTrigger">
          <item>
           <property name="text">
            <string>new content</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>file changes</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Time to display</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edit_TTL">
          <property name="maximumSize">
           <size>
            <width>35</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>seconds</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Presentation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Display on Monitor</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QRadioButton" name="radio_Monitor1">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="Dashboard.qrc">
               <normaloff>:/images/Monitor1.png</normaloff>:/images/Monitor1.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radio_Monitor2">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="Dashboard.qrc">
               <normaloff>:/images/Monitor2.png</normaloff>:/images/Monitor2.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radio_Monitor3">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="Dashboard.qrc">
               <normaloff>:/images/Monitor3.png</normaloff>:/images/Monitor3.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radio_Monitor4">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="Dashboard.qrc">
               <normaloff>:/images/Monitor4.png</normaloff>:/images/Monitor4.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Animation</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Entry type</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="combo_EntryType">
             <item>
              <property name="text">
               <string>SlideDownLeftTop</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>SlideDownCenterTop</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>SlideDownRightTop</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>SlideInLeftTop</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>SlideInRightTop</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>SlideInLeftBottom</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>SlideInRightBottom</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>SlideUpLeftBottom</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>SlideUpRightBottom</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>SlideUpCenterBottom</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>FadeInCenter</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>PopCenter</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>PopLeftTop</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>PopRightTop</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>PopLeftBottom</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>PopRightBottom</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Exit type</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="combo_ExitType">
             <item>
              <property name="text">
               <string>Pop</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Fade</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>SlideLeft</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>SlideRight</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>SlideUp</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>SlideDown</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>SlideFadeLeft</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>SlideFadeRight</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>SlideFadeUp</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>SlideFadeDown</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Reports should</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QRadioButton" name="radio_Stacked">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Reports for an entity will appear in their own lane, stacked report lanes for other entities.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Stack</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radio_Intermixed">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Reports for entities will be intermixed in a single lane.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Intermix</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Dashboard.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddLocalDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddLocalDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
